### PR TITLE
Added example for Fade transform

### PR DESCRIPTION
**Pull request by radioactive11**
Added Example for [Fade Transform](https://pytorch.org/audio/stable/transforms.html#torchaudio.transforms.Fade) as mentioned in [this issue](https://github.com/pytorch/audio/issues/1564).

### CODE REVIEW

File: torchaudio/transforms.py [hunk 1]
```diff
@@ -996,6 +996,11 @@ class Fade(torch.nn.Module):
         fade_out_len (int, optional): Length of fade-out (time frames). (Default: ``0``)
         fade_shape (str, optional): Shape of fade. Must be one of: "quarter_sine",
             "half_sine", "linear", "logarithmic", "exponential". (Default: ``"linear"``)
+            
+     Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+        >>> transform = transforms.Fade(fade_in_len=1, fade_out_len=2, fade_shape='linear')
```
Comment (nateanl) [inline torchaudio/transforms.py:1002]: 1 and 2 sample points seem too few. Could you change it to 1 second and 2 seconds? 
```suggestion
        >>> transform = transforms.Fade(fade_in_len=sample_rate, fade_out_len=2 * sample_rate, fade_shape='linear')
```

**Review by nateanl — CHANGES_REQUESTED** [review]

File: torchaudio/transforms.py (hunk 1 above)
Comment (radioactive11) [reply torchaudio/transforms.py]: Surely!

**Review by nateanl — APPROVED** [review]
LGTM, thanks for the PR!

File: torchaudio/transforms.py [hunk 2]
```diff
@@ -996,6 +996,11 @@ class Fade(torch.nn.Module):
         fade_out_len (int, optional): Length of fade-out (time frames). (Default: ``0``)
         fade_shape (str, optional): Shape of fade. Must be one of: "quarter_sine",
             "half_sine", "linear", "logarithmic", "exponential". (Default: ``"linear"``)
+            
+     Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+        >>> transform = transforms.Fade(fade_in_len=sample_rate, fade_out_len=2 * sample_rate, fade_shape='linear')
+        >>> faded_waveform = transform(waveform)
```
Comment (nateanl) [inline torchaudio/transforms.py:1003]:
```suggestion

    Example
        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
        >>> transform = transforms.Fade(fade_in_len=sample_rate, fade_out_len=2 * sample_rate, fade_shape='linear')
        >>> faded_waveform = transform(waveform)
```

**Pull request merged**